test/test_synthetic.py

```diff
@@ -193,7 +191,7 @@ def test_dl_real(region, mod='ERA5'):
     proc = subprocess.run(cmd.split(), stdout=subprocess.PIPE, universal_newlines=True)
     assert proc.returncode == 0, 'RAiDER did not complete successfully'
 
-
+@pytest.mark.long()
```

Comment:
@jlmaurer can we remove these long? The weather models are staged so they take like 2 min each max

Reply:
@bbuzz31 these take forever on my system...are they staged here?